docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: "3"
```

Comment:
I recommend you to check the Dockerfile in the backend project to see how we usually do it in Travelperk

Reply:
Changed to be consistent with docker-compose version used in backend project.